istioctl/cmd/describe.go

```diff
@@ -671,6 +684,64 @@ func (v *myGogoValue) keyAsString(key string) string {
 	return s.GetStringValue()
 }
 
+func getIstioRBACPolicies(cd *configdump.Wrapper, port int32) ([]string, error) {
+	hcm, err := getInboundHTTPConnectionManager(cd, port)
+	if err != nil || hcm == nil {
+		return []string{}, err
+	}
+
+	// Identify RBAC policies.  Currently there are no "breadcrumbs" so we only
+	// return the policy names, not the ServiceRole and ServiceRoleBinding names.
+	for _, httpFilter := range hcm.HttpFilters {
+		if httpFilter.Name == authz_model.RBACHTTPFilterName {
+			rbac := &rbac_http_filter.RBAC{}
+			if err := gogo_types.UnmarshalAny(httpFilter.GetTypedConfig(), rbac); err == nil {
+				policies := []string{}
+				for polName := range rbac.Rules.Policies {
+					policies = append(policies, polName)
+				}
+				return policies, nil
+			}
+		}
+	}
+
+	return []string{}, nil
+}
+
+// Return the first HTTP Connection Manager config for the inbound port
+func getInboundHTTPConnectionManager(cd *configdump.Wrapper, port int32) (*http_conn.HttpConnectionManager, error) {
+	filter := istio_envoy_configdump.ListenerFilter{
+		Port: uint32(port),
+	}
+	listeners, err := cd.GetListenerConfigDump()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, listener := range listeners.DynamicActiveListeners {
+		if filter.Verify(listener.Listener) {
+			sockAddr := listener.Listener.Address.GetSocketAddress()
+			if sockAddr != nil {
+				// Skip outbound listeners
+				if sockAddr.Address == "0.0.0.0" {
+					continue
+				}
+			}
+
+			for _, filterChain := range listener.Listener.FilterChains {
+				for _, filter := range filterChain.Filters {
```

Comment:
We always use the 1st Filter in the 1st FilterChain? Any case we should use the other one?